static/app/views/issueDetails/traceDataSection.spec.tsx

```diff
@@ -212,7 +213,13 @@ describe('TraceDataSection', () => {
       body: [],
     });
 
-    render(<TraceDataSection event={event} />, {organization});
+    // Use SurfaceProvider to test the old analytics event is still emitted
```

Comment:
aren't we using `SurfaceProvider` here to test that the new analytics event, the one with `surface`, is being emitted?

Reply:
We're using it for both!